src/main/kotlin/lotto/model/LottoMachine.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.model
+
+import camp.nextstep.edu.missionutils.Randoms
+
+object LottoMachine {
+
+    fun issueLottos(purchaseAmount: PurchaseAmount): List<Lotto> {
+        val lottoCount = purchaseAmount.getLottoCount()
+        val lottos = mutableListOf<Lotto>()
+        repeat(lottoCount) {
+            lottos.add(
+                Lotto(Randoms.pickUniqueNumbersInRange(LottoNumber.MIN_NUMBER, LottoNumber.MAX_NUMBER, Lotto.LOTTO_SIZE))
+            )
+        }
```

Comment:
```suggestion
        val lottos = List(lottoCount) {
           Lotto(Randoms.pickUniqueNumbersInRange(LottoNumber.MIN_NUMBER, LottoNumber.MAX_NUMBER, Lotto.LOTTO_SIZE))
            )
        }
```

List() 팩토리 함수를 사용하면 더 간단하게 나타낼 수 있습니다!